praw/__init__.py

```diff
@@ -1610,9 +1610,8 @@ def set_settings(self, subreddit, title, public_description='',
                      submit_link_label='', submit_text_label='',
                      exclude_banned_modqueue=False, comment_score_hide_mins=0,
                      public_traffic=False, collapse_deleted_comments=False,
-                     spam_comments='low', spam_links='high',
-                     spam_selfposts='high', submit_text='',
-                     hide_ads=False, **kwargs):
+                     spam_comments='low', spam_links='high', spam_selfposts='high',
+                     submit_text='', hide_ads=False, suggested_comment_sort, **kwargs):
```

Comment:
Shouldn't it be `suggested_comment_sort=None`? This way it's not a mandatory field.

`None or ''` below will make it '', which will then make the sort the default of actually None via [here](https://github.com/reddit/reddit/blob/master/r2/r2/controllers/api.py#L2674)